src/Containers/Pages/IngredientSelection/IngredientSelection.js

```diff
@@ -1,108 +1,142 @@
-                </Grid>
-            </div>
-            <div className={classes.container}>
-                <Grid container className={classes.header}>
-                    <img item className={classes.photo} src={produceImage} alt='produce logo'/>
-                    <h2 item className={classes.header}>Produce</h2>
-                    <OptionChips item options={produceOptions} setOptions = {setProduce}/>
-                </Grid>
-            </div>
-            <div className={classes.container}>
-                <Grid container className={classes.header}>
-                    <img item className={classes.photo} src={bakingImage} alt='baking logo'/>
-                    <h2 item className={classes.header}>Baking & Grains</h2>
-                    <OptionChips item options={bakingOptions} setOptions = {setBaking}/>
-                </Grid>
-            </div>
-            <Grid container justify="center">
-                <Button item className={classes.button} 
-                    onClick = {() => {}}>Next
-                </Button>
-            </Grid>
-        </>
-    )
-}
-export default withRouter(IngredientSelection);
\ No newline at end of file
+  return (
+    <>
+      <div>
+        <h1 className={classes.title}>What ingredients do you have?</h1>
+        <p className={classes.text}>(pick at least 5)</p>
+      </div>
+      <div className={classes.container}>
+        <Grid container className={classes.header}>
+          <img
+            item
+            className={classes.photo}
+            src={dairyImage}
+            alt='dairy logo'
+          />
+          <h2 item className={classes.header}>
+            Dairy
+          </h2>
+          <OptionChips item options={dairyOptions} setOptions={setDairy} />
+        </Grid>
+      </div>
+      <div className={classes.container}>
+        <Grid container className={classes.header}>
+          <img
+            item
+            className={classes.photo}
+            src={produceImage}
+            alt='produce logo'
+          />
+          <h2 item className={classes.header}>
+            Produce
+          </h2>
+          <OptionChips item options={produceOptions} setOptions={setProduce} />
+        </Grid>
+      </div>
+      <div className={classes.container}>
+        <Grid container className={classes.header}>
+          <img
+            item
+            className={classes.photo}
+            src={bakingImage}
+            alt='baking logo'
+          />
+          <h2 item className={classes.header}>
+            Baking & Grains
+          </h2>
+          <OptionChips item options={bakingOptions} setOptions={setBaking} />
+        </Grid>
+      </div>
+      <Grid container justify='center'>
+        <Button
+          item
+          className={classes.button}
+          onClick={() => {
+            addDairyOption(filterItem(dairyOptions));
+            addProduceOption(filterItem(produceOptions));
+            addBakingOption(filterItem(bakingOptions));
```

Comment:
does this really need to be stored in different nodes? It just ingredient lists basically.